setup.py

```diff
@@ -182,7 +182,7 @@ def check_for_openmp():
 
         "extras_require": {':python_version=="2.6"': ['argparse>=1.2.1'],
                            'docs': ['sphinx', 'sphinxcontrib-autoprogram'],
-                           'tests': ['nose >= 1.0']},
+                           'tests': ['nose >= 1.0', 'pysam']},
```

Comment:
I think this is a heavy requirement that is suitable for sandbox but we should rethink for longer term, & certainly prior to merge.  There was that replacement that came across twitter today that we should look at.